model.py

```diff
@@ -208,14 +249,25 @@ def decode_image_latents(self, denoised_latent_image: Tensor) -> np.ndarray:
 
 # Singular forward pass with fake images
 # (B, 3, I, I)
-def forward_pass_test():
-    img = torch.randn([BATCH_SIZE, 3, IMG_DIM, IMG_DIM], device=DEVICE)
-    input_str = """
-    Once upon a time, in the heart of an ancient kingdom, there was a stunning garden filled with vibrant flowers and chirping birds. Amidst the garden, a glistening pond served as a mirror, reflecting the blossoming surroundings. 
-    A gentle breeze rustled the leaves of the towering trees, adding a melodious charm to the scene. The garden was indeed a paradise, a spectacle of nature's magnificence, where every creature lived in har
-    """
-    denoised_latent_img = model(img, [input_str]*BATCH_SIZE, [1]*BATCH_SIZE)
-    # print(denoised_latent_img.shape)
+@torch.no_grad()
+def forward_pass_test(model: LLourney, input_str: str, batch_size: int = 2, device="cpu"):
+    # Prepare random image in pixel space
+    img = torch.randn((batch_size, 3, model.image_dim, model.image_dim), device=device)
```

Comment:
to make this a FloatTensor, could add `.double()` here to make the types line up

Reply:
I'm not sure if I understand the comment, [`torch.randn`](https://pytorch.org/docs/stable/generated/torch.randn.html) should use `dtype = torch.float (= torch.float32)` by default. So it's already a `torch.FloatTensor`, and I'm not sure casting it to a [double](https://pytorch.org/docs/stable/generated/torch.Tensor.double.html) (= `torch.float64`) would make sense here.

Reply:
Basically Pycharm complains at the typing (it doesn't believe `img` is a FloatTensor) which could be its own fault. But adding `.double()` suppresses that error. Not a big deal in any case.